assets/build/install.sh

```diff
@@ -210,6 +210,7 @@ sed -i \
   -e "s|^[#]*UsePrivilegeSeparation yes|UsePrivilegeSeparation no|" \
   -e "s|^[#]*PasswordAuthentication yes|PasswordAuthentication no|" \
   -e "s|^[#]*LogLevel INFO|LogLevel VERBOSE|" \
+  -e "s|^[#]*AuthorizedKeysFile.*|AuthorizedKeysFile %h/.ssh/authorized_keys %h/.ssh/authorized_keys_proxy|" \
```

Comment:
> Probably we don't to add this to the install.sh

I have not noticed `%h/.ssh/authorized_keys_proxy` undesirable side effects.

[omnibus GitLab Docker Image](https://gitlab.com/gitlab-org/omnibus-gitlab/commit/923fd761ed854ca368c413a581b1153cd677dbe5#d99621b50fd86a7df71d4fa8a395a3736e43fc90) have also this hack.

> And mount the config back inside the container because it was created during build and changed by the runtime

It was my first implementation but I can't use it, I had a error (I do not remember precisely the error name) 😞.

Do you want that I spend some time to retrieve the error?